### ISSUE TITLE
Template Fills

### ISSUE THREAD

**Issue by eighteyes** [warpdotdev/Warp#3172]
### Describe the solution you'd like?

```Text
When I do an AI generated command, I end up with a bunch of variables in {{brackets}}. Which I have to manually replace. It would be great if I could do a IDE snippet style replacement, where I hit a key ( like Tab, but that's overloaded ), that selects the variable, lets me type in a replacement and I can hit it again to go to the next one. 

A little efficiency, but it would go a long way, especially combined with your private terminal knowledge base ( and pre-loaded templates )
```

### Is your feature request related to a problem? Please describe.

```Text
Moving from mouse to keyboard is marginally less painful then holding down arrow keys to repeat.
```

### How important is this feature to you?

3

### Warp Internal (ignore) - linear-label:770f6576-d6c0-4e4f-a259-fc64b5156087

None

**Comment by dannyneira** [comment]
Thanks for this feature request! This is a cool idea and totally something that would make sense to combine [Warp AI](https://docs.warp.dev/features/warp-ai) with [Workflows](https://docs.warp.dev/features/entry/workflows) to easily create parameterized snippets.

Please add a :+1: to the original post at the top to signal that you want this feature, and then hit the subscribe button if you'd like to be notified.

**Comment by eighteyes** [comment]
Exactly, I knew I had seen that feature in Warp and was boggled that it didn't work as I had remembered.

**Comment by lfesp** [comment]
This should be fixed with release v0.2023.07.11.08.03 as a feature of the new AI Command Search experience, please comment if it's not working as intended!

**Comment by dannyneira** [comment]
Closing as resolved!
